webdriver/tests/bidi/script/realm_created/realm_created.py

```diff
@@ -363,3 +374,28 @@ async def on_event(method, data):
         },
         realm,
     )
+
+
+@pytest.mark.parametrize("type_hint", ["tab", "window"])
+async def test_existing_realm(bidi_session, wait_for_event, wait_for_future_safe, subscribe_events, test_origin, type_hint):
```

Comment:
Shouldn't we also include a test for frames, and one more for already created sandboxes?

Reply:
We should. But I a not sure if I will have time to spend on this, so will merge at least this for now.

Reply:
@Lightning00Blade are you going to merge?

Reply:
@whimboo The `wpt-firefox-nightly-stability ` and I can't merge normally I think we should force merge?

Reply:
Ah, so `realm_created.py` times out because we may have too many long running tests in that file. You should probably extend the timeout, or splitting the file.

Reply:
If I understand correctly how WPT work is if we increase the timeout that will increase the Timeout multiplier and then `wait_for_future_safe` will wait for even longer, which will not change what we are seeing now.
Or the other option is to as you said split the test.

Reply:
I think the suggestion is just to add `# META: timeout=long` which doesn't affect the multiplier.